### PR TITLE
Update Intake.java

### PR DESCRIPTION
**Pull request by ILIK3T0C0D3**
This adds one more IntakeMotor

### CODE REVIEW

**Review by SamueljWilson — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
 
   public static class IntakeConstants {
     //TODO: Assign a real CAN ID
-    public static final int kIntakeMotorID = 0;
+    public static final int kTopIntakeMotorID = 0;
+    public static final int kBottomIntakeMotorID = 1;
     public static final int kIntakeCurrentLimit = 40;
+    public static final double kShaftDiameter = 0.0127;
```
Comment (jasone) [inline src/main/java/frc/robot/Constants.java]: By shaft diameter, do you mean intake wheel diameter? That's `Units.inchesToMeters(1.0)` for the bottom intake roller, and `Units.inchesToMeters(2.0)` for the top intake roller.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
...
+    public static final double kTopGearRatio = 0.83;
```
Comment (jasone) [inline src/main/java/frc/robot/Constants.java]: This is hard to understand/audit. `15 / 18` would be clearer.

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
...
+    public static final double kBottomGearRatio = 1;
+    public static final double kTopVelocityConversionFactor = (kShaftDiameter * Math.PI)/kTopGearRatio/60;
+    public static final double kBottomVelocityConversionFactor = (kShaftDiameter * Math.PI)/kBottomGearRatio/60;
```
Comment (jasone) [inline src/main/java/frc/robot/Constants.java]: The only way I can readily tell these are correct is by looking at e.g. `kDriveVelocityConversionFactor` for comparison. At a minimum there should be a comment saying what units these convert between.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 4]
```diff
@@ -5,41 +5,74 @@
...
+    m_intakePidControllerBottom.setFeedbackDevice(m_intakeEncoderBottom);
+    m_intakePidControllerBottom.setP(IntakeConstants.kIntakePIDBottom.p(), 0);
+    m_intakePidControllerBottom.setI(IntakeConstants.kIntakePIDBottom.i(), 0);
+    m_intakePidControllerBottom.setD(IntakeConstants.kIntakePIDBottom.d(), 0);
+    m_intakePidControllerBottom.setFF(0);
 
     m_ultrasonicSensor = new Ultrasonic(pingPort, echoPort);
   }
 
   public void startIntake(double speed) {
```
Comment (jasone) [inline src/main/java/frc/robot/subsystems/Intake.java:56]: What are the units for `speed`?

**Review by jasone — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 4 above)
Comment (ILIK3T0C0D3) [reply src/main/java/frc/robot/subsystems/Intake.java:56]: They are meters

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (ILIK3T0C0D3) [reply src/main/java/frc/robot/Constants.java]: This is fixed now

File: src/main/java/frc/robot/Constants.java (hunk 3 above)
Comment (ILIK3T0C0D3) [reply src/main/java/frc/robot/Constants.java]: I made the comment, and it is now on line 207 in constants.java

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (ILIK3T0C0D3) [reply src/main/java/frc/robot/Constants.java]: I made new commands that replace ShaftDiameter. I don't know where we got the number for it but the intake wheel diameter number is the correct number for the calculation ShaftDiameter was going to perform.

File: src/main/java/frc/robot/Constants.java [hunk 5]
```diff
@@ -201,12 +201,13 @@ public static class IntakeConstants {
     public static final int kTopIntakeMotorID = 0;
     public static final int kBottomIntakeMotorID = 1;
     public static final int kIntakeCurrentLimit = 40;
-    public static final double kShaftDiameter = 0.0127;
+    public static final double kTopIntakeWheelDiameter = 2;
+    public static final double kBottomIntakeWheelDiameter = 1;
```
Comment (jasone) [inline src/main/java/frc/robot/Constants.java]: These have to be e.g. `Units.inchesToMeters(2)`.

**Review by jasone — CHANGES_REQUESTED** [review]

**Review by jasone — APPROVED** [review]
👍

**Pull request merged**